test/system/030-run.bats

```diff
@@ -305,4 +305,22 @@ echo $rand        |   0 | $rand
     run_podman wait $cid
 }
 
+@test "podman run --log-driver" {
+    run_podman run --log-driver k8s-file --log-opt path=/var/log/podmantest.log --rm $IMAGE echo "podman log-driver test"
+    run grep "podman log-driver test" /var/log/podmantest.log
+    rm -f /var/log/podmantest.log
+    is "$output" ".*stdout.*podman log-driver test" "log should exist in log file"
+
+    run_podman run --log-driver journald --rm $IMAGE echo "podman log-driver test"
+    run grep  "podman log-driver test" <(journalctl)
+    is "$output" ".*podman log-driver test" "log should exist in journalctl output"
```

Comment:
Could you also add the podman logs test here, to make sure we can read the data back from the journal via the comand.